app/main/views/briefs.py

```diff
@@ -151,12 +149,39 @@ def update_brief_add_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['PUT'])
+def update_brief_add_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    # Using the helper.py methods to get the matching brief.
+    brief = briefs.get(brief_id)
+    supplier = suppliers.get_supplier_by_code(supplier_code)
+
+    if not supplier:
+        raise ValidationError("No supplier found: " + supplier_code)
+    sellers = brief.data['sellers']
+    sellers[str(supplier_code)] = {'name': supplier.name}
+    brief.data['sellers'] = sellers
+    audit = AuditEvent(
+        audit_type=AuditTypes.seller_added_to_rfx_opportunity_admin,
+        user=updater_json['updated_by'],
+        data={
+            'briefId': brief.id,
+            'supplier_code': supplier_code,
+        },
+        db_object=brief,
+    )
+
+    db.session.add(brief)
+    db.session.add(audit)
+    db.session.commit()
+
+    return jsonify(briefs=brief.serialize(with_users=True)), 200
+
+
 @main.route('/briefs/<int:brief_id>/users/<string:user>', methods=['DELETE'])
 def update_brief_remove_user(brief_id, user):
     updater_json = validate_and_return_updater_request()
-    brief = Brief.query.filter(
-        Brief.id == brief_id
-    ).first_or_404()
+    brief = briefs.get(brief_id)
```

Comment:
not sure why you updated only this one?  maybe revert since nothing else has changed in this function?